etl/steps/data/garden/health/2023-04-18/shared.py

```diff
@@ -0,0 +1,688 @@
+"""Utils (mostly mappings)
+
+These are manually created, based on:
+
+- "Guide to using the data": https://cms.wellcome.org/sites/default/files/2021-10/wgmdata-mental-health-data-dictionary-user-guide.docx (provides mapping of answer IDs to answer labels, only for MH* fields)
+- "Full questionnaire":
+    - WGM mental health 2020: https://cms.wellcome.org/sites/default/files/2021-11/WGM_Full_Questionnaire.pdf (based on order of answers, we manually map answer IDs to answer labels. Ideally WGM would provide a dictionary)
+    - Need to complement with https://wellcome.org/reports/wellcome-global-monitor-covid-19/2020 (for the W* questions)
+
+"""
+
+
+MAPPING_GENDER_VALUES = {
+    1: "male",
+    2: "female",
+}
+MAPPING_AGE_VALUES = {
+    1: "15-29",
+    2: "30-49",
+    3: "50-64",
+    4: "65+",
+    99: "DK/Refused",
+}
+# Rename fields
+MAPPING_COLUMN_NAMES = {
+    "countrynew": "country",
+    "wgt": "weight_intra_country",
+    "projwt": "weight_inter_country",
+    "year_wave": "year",
+    "gender": "gender",
+    "age_var2": "age_group",
+}
+# Extracted from
+# - https://cms.wellcome.org/sites/default/files/2021-11/WGM_Full_Questionnaire.pdf (full questionaire)
+# - https://cms.wellcome.org/sites/default/files/2021-10/wgm-mentalhealthmodule-crossnational-tabs.xlsx (data report, xlsx)
+MAPPING_QUESTION_VALUES = {
+    "mh1": {
+        "title": "Importance of mental health for well-being",
+        "answers": {
+            "1": "More important",
+            "2": "As important",
+            "3": "Less important",
+            "99": "DK/Refused",
+        },
+    },
+    "mh2a": {
+        "title": "How much science can explain how the human body works",
+        "answers": {
+            "1": "A lot",
+            "2": "Some",
+            "3": "Not much",
+            "4": "Not at all",
+            "99": "DK/Refused",
+        },
+    },
+    "mh2b": {
+        "title": "How much science can explain how feelings and emotions work",
+        "answers": {
+            "1": "A lot",
+            "2": "Some",
+            "3": "Not much",
+            "4": "Not at all",
+            "99": "DK/Refused",
+        },
+    },
+    "mh3a": {
+        "title": "How much science helps to treat cancer",
```

Comment:
I guess "How much does science help [...] ?" would sound more natural, but maybe in indirect form it's correct.

Reply:
I think it is fine to have the questions in indirect form